### DISCUSSION TITLE
SCRUM PREP MILESTONE 2-1

### DISCUSSION THREAD

**Discussion by Keiven-Soto** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#170] [General]
This week I met with the other manager and discussed the guidelines for milestone 2. Discussed new leadership appointments and  informed the team about the goals to be accomplished in this milestone 2.

**Comment by fernandopizarro1** [comment]
Started with a more complex and optimal version of the domino bot, where it can play a tile depending on the chances to win or to block the game, also keeping track of the tiles already played.

**Comment by sandy-marrero** [comment]
Worked on integrating the login form with the backend, made minor change to the backend. Will continue work on the profile component of the application. Will work on adding additional error handling to login. #131

**Comment by AnthonyMed13** [comment]
Started fixing some coding errors with the JWT secret key of the login and logout end points.

**Comment by PinkSylvie** [comment]
This week I created tasks to hold over the decay on the fixes that will need to be done on the documentation and started to research on the class material to work on the tasks I picked regarding the documentation.

**Comment by ebrain-mendoza121** [comment]
This week attended meetings with managers, made tasks to expand lobby features and implementation.

**Comment by print-Eruki** [comment]
Reviewed pull request from last milestones, causing errors in main. Work alongside morph_ina with UUID integration to backend.

**Comment by RicardoJRSs** [comment]
This week I attended meetings to design the lobby system for the game and integrate it

**Comment by JadiellMV** [comment]
This week I created the tasks I will be working on and also started working on them, specifically on error handling when it comes to wrong password on login and already registered users on sign up.

**Comment by AngelCIICMorales** [comment]
Created milestone guidelines, held meeting with the team in order to jumpstart tasks.

**Comment by Gaby-Concepcion** [comment]
Milestone 2, Week 1
So far this week I have mostly worked on clearing up and expanding on certain areas of documentation in order to include a broader view of upcoming features that we will incorporate during this milestone. I have also set time aside from the project as our first midterm for the class is coming up. Moving towards week 2 I hope to be done with any exams by Wednesday and start really cracking down on any of my more complex issues then.

**Comment by nsfrontera** [comment]
Worked with the team lead to discuss the task i would create  and selected tasks, and will be working on creating lecture topic tasks

**Comment by ilikecheese117** [comment]
This week I assigned a task related to save system.

**Comment by G4vp** [comment]
Reviewing issues and planning to create lecture topic tasks.

**Comment by Negroni1** [comment]
Work alongside @print-Eruki with UUID integration to backend and review game logic for multiplayer integration, to be able to have a proper web socket server

**Comment by Adriana-Isabel** [comment]
I started working on the Header and will probably finish it by Monday or Tuesday. The Header will include a home button, an instructions button, Frequently Asked Questions button and a Sign Up/Log in button.

**Comment by Jvstinoo** [comment]
This week I assigned myself tasks. Also, started creating lecture topic task.

**Comment by DavidCuevas3** [comment]
Created the task for implementing the multiplayer leaderboard into the project and will complete this issue in the following weeks.

**Comment by Saul-Caban-UPRM** [comment]
Created two task #157 and #164, researched how to implement assets into the game

**Comment by FrancesC8** [comment]
Completed tasks from mileston 1 and created new tasks for milestone 2. Haven't started any work on the currently assigned tasks.

**Comment by ianramospr** [comment]
I've been formulating/figuring out my tasks for this Milestone and will upload them today.

**Comment by Joaquin939** [comment]
This week I started looking for some issues for me to work on for this milestone

**Comment by rukitoyordan** [comment]
This week I met up with @frances-sepulveda to discuss what LTTs and distribution management on new powerups for the avatars and board.
